odonto/odonto_submissions/tests/test_serializors.py

```diff
@@ -165,7 +165,7 @@ def test_without_ethnicity(self):
         with self.assertRaises(serializers.SerializerValidationError) as e:
             serializers.DemographicsTranslator(self.demographics).ethnicity()
         self.assertEqual(
-            str(e.exception), f"Unable to find an ethnicity for patient {self.demographics.patient_id}"
```

Comment:
This kind of churn is why I rarely explicitly test error message strings - does it raise an error, is it the right class is generally enough IMO /shrug